tests/MaintainersTest.php

```diff
@@ -258,86 +258,6 @@ public function testSettingAutoJoinFlag()
         $this->assertEquals(false, $org['allowAutoJoin']);
     }
 
-    /**
-     * @dataProvider autoJoinProvider
-     * @param bool $allowAutoJoin
-     */
-    public function testMaintainerAutoJoinError(bool $allowAutoJoin)
```

Comment:
Nahrazeno `ProjectJoinTest::testSuperAdminCannotAddSelfToProjectRegardlessOfAllowAutoJoin()`

Tady to imho nemelo byt.